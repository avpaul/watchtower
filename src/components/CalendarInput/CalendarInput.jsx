import React from 'react';

const CalendarInput = () => (
  <div className="input-group-apend text-right">
    <div className="input-group-duration ">
      <i className="fas fa-calendar-alt" />
    </div>
  </div>
);

export default CalendarInput;
