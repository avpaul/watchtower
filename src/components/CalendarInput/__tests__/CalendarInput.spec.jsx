import React from 'react';
import { shallow } from 'enzyme';
import CalendarInput from '../CalendarInput';

describe('Test CalendarInput snapshots', () => {
  it('CalendarInput should render properly', () => {
    expect(shallow(<CalendarInput />)).toMatchSnapshot();
  });
});
