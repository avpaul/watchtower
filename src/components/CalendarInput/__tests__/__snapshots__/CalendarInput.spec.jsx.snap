// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test CalendarInput snapshots CalendarInput should render properly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CalendarInput />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="input-group-duration "
>
        <i
                className="fas fa-calendar-alt"
        />
</div>,
      "className": "input-group-apend text-right",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <i
          className="fas fa-calendar-alt"
/>,
        "className": "input-group-duration ",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "fas fa-calendar-alt",
        },
        "ref": null,
        "rendered": null,
        "type": "i",
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="input-group-duration "
>
          <i
                    className="fas fa-calendar-alt"
          />
</div>,
        "className": "input-group-apend text-right",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <i
            className="fas fa-calendar-alt"
/>,
          "className": "input-group-duration ",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "fas fa-calendar-alt",
          },
          "ref": null,
          "rendered": null,
          "type": "i",
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
}
`;
